README.md

```diff
@@ -2,9 +2,36 @@
 
 ### Introduction
 
-*Read the full introduction on my [Medium blog](https://medium.com/arjunkalburgi/theradlist-a-new-side-product-e07bf0d3fdd8)*
+TheRadList is a tool to help Edmontonians search through restaurants, bars and cafes in their city with useful filters. 
+When you're looking for a restaurant, TheRadList is the place to look. 
+
+I started this project in January of 2018. Every once in a while I’ll bring it back to update it or use it to learn something new.
+
+Here’s the total history of TheRadList updates:
+
+* [Initial launch of the project](https://medium.com/arjunkalburgi/theradlist-a-new-side-product-e07bf0d3fdd8)
+* [Update to learn Angular 5](https://github.com/theradlist/angular-radlist)
+* [Update so that my friends can easily modify content](https://medium.com/arjunkalburgi/im-more-lazy-than-you-backends-with-google-spreadsheets-3eeac8d135c1)
+* [Update to learn Vue.js](https://medium.com/arjunkalburgi/viewing-vue-with-theradlist-8851502ab709)
+
+
+### Frontend Details 
+
+The most recent update of this project was refactoring the app to use Vue.js. 
+
+I write about that [here](https://medium.com/arjunkalburgi/viewing-vue-with-theradlist-8851502ab709). 
+
+I originally made this website with a little bit of copying from [The Ringers 2018 NBA Draft Guide](https://nbadraft.theringer.com/) and hacked from there. A lot of the CSS is still from there, which is why it's a little unorganized. 
+
+
+### Backend Details
+
+I use Google Spreadsheets with the [Sheety API](https://sheety.co/) as my backend. 
+
+Further details [here](https://medium.com/arjunkalburgi/im-more-lazy-than-you-backends-with-google-spreadsheets-3eeac8d135c1) and you can view the data [here](https://www.theradlist.com/data).
+
+Because no Front End Dev really wants to mess with a backend :P 
```

Comment:
While I enjoy this comment, you may want to remove it to be more professional :P